docs/source/research/bibliography/ODEs.bib

```diff
@@ -328,14 +328,13 @@ @incollection{teymur18
   file = {https://arxiv.org/pdf/1805.07970.pdf}
 }
 
-@article{breen2019bulirsch,
```

Comment:
I think this is indeed ok to remove. The authors originally contacted us to have this added, but now that we're slimming the list, I agree that it's not a crucial thing to have around.

Reply:
Ah, I actually removed this because it's been superseded by https://arxiv.org/abs/2106.13718 , which I've added to "General" :-)

Reply:
Even better! Thanks!